velox/exec/tests/ExchangeClientTest.cpp

```diff
@@ -474,5 +473,143 @@ TEST_F(ExchangeClientTest, callNextAfterClose) {
   client->close();
 }
 
+TEST_F(ExchangeClientTest, acknowledge) {
```

Comment:
nit: acknowledge -> pause ?

Reply:
I thought about renaming it. But then I realized that it may not be accurate, as the test was valid even before the "pause" method was introduced (I introduce the test in a separate commit).

The ExchangeClientTest is implemented based on the interactions with OutputBuffer (via a LocalExchangeSource). This test verifies that `OutputBuffer` data is released as expected based on the interactions with `ExchangeClient`.